src/ecdsa/test_ecdsa.py

```diff
@@ -60,6 +60,57 @@ def test_rejection(self):
         assert not self.pubk.verifies(self.msg - 1, self.sig)
 
 
+class TestPublicKey(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+        
+    def test_equality_public_keys(self):
+        gen = generator_192
+        x = 0xc58d61f88d905293bcd4cd0080bcb1b7f811f2ffa41979f6
+        y = 0x8804dc7a7c4c7f8b5d437f5156f3312ca7d6de8a0e11867f        
+        point = ellipticcurve.Point(gen.curve(), x, y)
+        pub_key1 = Public_key(gen, point)
+        pub_key2 = Public_key(gen, point)
+        self.assertEqual(pub_key1, pub_key2)
+        
+    def test_inequality_public_key(self):
+        gen = generator_192
+        x1 = 0xc58d61f88d905293bcd4cd0080bcb1b7f811f2ffa41979f6
+        y1 = 0x8804dc7a7c4c7f8b5d437f5156f3312ca7d6de8a0e11867f  
+        point1 = ellipticcurve.Point(gen.curve(), x1, y1)
+                
+        x2 = 0x6a223d00bd22c52833409a163e057e5b5da1def2a197dd15
+        y2 = 0x7b482604199367f1f303f9ef627f922f97023e90eae08abf
+        point2 = ellipticcurve.Point(gen.curve(), x2, y2)
+        
+        pub_key1 = Public_key(gen, point1)
+        pub_key2 = Public_key(gen, point2)
```

Comment:
it's testing `Public_key` object defined in `ecdsa` module, shouldn't this test case be in `test_ecdsa.py` then?

also, it would be nice to have test coverage for the NotImplemented lines, comparing those object with `None` or a string should be enough